tests/ha/remove_node.pm

```diff
@@ -16,11 +16,12 @@ use warnings;
 use testapi;
 use lockapi;
 use hacluster;
+use version_utils 'is_sle';
 
 sub remove_state_join {
     my ($method, $cluster_name, $node_01, $node_02) = @_;
-    my $remove_cmd = 'crm cluster remove -y -c';
```

Comment:
Don't remove `crm cluster` for all version, as this should be the "new" command to use now.

Try this:

    my $cmd        = is_sle('=12-sp2') ? 'ha-cluster-' : 'crm cluster ';
    my $remove_cmd = "${cmd}remove -y -c";
    my $join_cmd   = "${cmd}join -y -i " . get_var('SUT_NETDEVICE', 'eth0') . ' -c';

Reply:
I talked with developer (Yan) last week about that and he told me to use (when it's possible) bootstrap scripts instead of `crm command`.

Reply:
OK, in that case... As I said in private it's not what a dev told me during SLE15, but I don't remember who.